libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/notification/NotificationMetric.java

```diff
@@ -0,0 +1,77 @@
+/*
+ *  Copyright 2020 Verizon Media
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.yahoo.athenz.common.server.notification;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+public class NotificationMetric {
+
+    // Metric attributes as a list of flat arrays
+    private List<String[]> attributes;
+
+    public NotificationMetric(List<String[]> attributes) {
+        this.attributes = attributes;
+    }
+
+    public List<String[]> getAttributes() {
+        return attributes;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+
+        NotificationMetric that = (NotificationMetric) o;
+
+        if (attributes.size() != ((NotificationMetric) o).attributes.size()) {
+            return false;
```

Comment:
NotificationMetric holds attributes in the data structure List<String[]> (a list of flat array metric records).
So I had to implement custom equals and hashCode (for tests to check equality correctly and to enable holding this structure in a hashmap if we would like to do so in the future).